### DISCUSSION TITLE
Fluxos de RealTokenizado e TPFt com Anonymous Zether

### DISCUSSION THREAD

**Discussion by LuisZapparolli** [bacen/pilotord-kit-onboarding#67] [Q&A] [answered]
Recentemente foi publicada a documentação a respeito dos fluxos anonimizados utilizando o Zether, porém apenas a manipulação de RealDigital está descrita no documento.

Está previsto que haverá esse único fluxo anonimizado, ou é uma prova de conceito e os demais contratos também irão prever fluxos anônimos?
Outra questão é, uma vez que a ferramenta esteja validada, ela substituirá os fluxos atuais ou irão coexistir e cada participante terá a escolha sobre utilizar ou não o Zether?

**Comment by aldenio** [comment, marked as answer]
Prezado @LuisZapparolli, 

Foram também disponibilizados fluxos Zether para títulos públicos e DvP, e todos os fluxos coexistirão durante o piloto-Drex.
Não há previsão de substituição dos fluxos atuais durante a fase piloto do Drex. 
Nossa orientação aos participantes é que testem todas as ferramentas implantadas e compartilhem conosco suas percepções e dificuldades sobre tais ferramentas.
